cmd/controller/main.go

```diff
@@ -43,6 +45,8 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.IntVar(&concurrentReconciles, "concurrent-reconciles", 10, "The concurrency level for reconciling resources.")
```

Comment:
should we default to a smaller value?

```suggestion
	flag.IntVar(&concurrentReconciles, "concurrent-reconciles", 1, "The concurrency level for reconciling resources.")
```

Reply:
nope, 1 is too low. we will need to make patches if infra-depl for every installation.
so i would left it as 10 but reduce where it is needed (so far ocp-art only)

Reply:
To keep same behavior as today, I would keep default to 10